crypto/x509/by_store.c

```diff
@@ -116,7 +116,11 @@ static int by_store_ctrl_ex(X509_LOOKUP *ctx, int cmd, const char *argp,
             argp = ossl_safe_getenv(X509_get_default_cert_dir_env());
 
         if (argp == NULL)
+#ifndef _WIN32
```

Comment:
Could you please make it positive instead of negative?

#ifdef _WIN32

it helps improve readability. Thanks!